iron-multi-selectable.html

```diff
@@ -108,6 +115,12 @@
         this.splice('selectedValues',i,1);
       }
       this._selection.setItemSelected(this._valueToItem(value), unselected);
+    },
+
+    _valuesToItems: function(values) {
+      return (values == null) ? null : values.map(function(value) {
```

Comment:
After looking around for a bit, it seems like there's still perf implications to using bind; I think we should switch this back to the `thisArg` approach. :/ Sorry for leading you astray there!